src/main/java/callhub/connect/drivers/WebSocketConfig.java

```diff
@@ -1,23 +1,26 @@
 package callhub.connect.drivers;
 
+import org.springframework.context.annotation.Configuration;
 import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
 import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
 import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
 
+@Configuration
 @EnableWebSocketMessageBroker
 public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
 
     @Override
     public void registerStompEndpoints(StompEndpointRegistry
                                                registry) {
         registry.addEndpoint("/mywebsockets")
-                .setAllowedOrigins("http://localhost:3000/").withSockJS();
+                .setAllowedOrigins("*");
```

Comment:
can we set the allowed origins to the deployed app and to the local host 3000 + 8000?

Reply:
sorry i meant 8080

Reply:
Good idea. But heads up, I changed the frontend local host port to 3000 because it was the same as the backend before (at 8080)

Reply:
done